### PR TITLE
works using timeline

### PR DESCRIPTION
**Pull request by IsaacA2151**

### CODE REVIEW

**Comment by IsaacA2151** [comment]
ignore 
he = Button(tab,text='hello',command=self.add_week)
he.grid(row=10,column=0)

File: panel_grid.py [hunk 1]
```diff
@@ -46,34 +55,38 @@ def create_panel(self):
         tab = Frame(tablayout)
         tab.pack(fill="both")
 
-        for i in range(6):              # for rows in activities
-            for j in range(8):          # for cols in days
+        for i in range(len(self.activities[self.week-1])+1):              # for rows in activities
```
Comment (midastown) [inline panel_grid.py:58]: from this line to line 76, needs to move into it's own function which would take "a" week in the timeline. Right now we don't have anything that will cache the Timeline object in the memory, so when we call create_panel() it's gonna be the first week, but once we already have "a" Timeline object in the memory, we could pass it the current week of today, whatever today may be.

File: panel_grid.py [hunk 2]
```diff
@@ -14,8 +15,16 @@ def __init__(self, window, heigth, width):
...
-        self.activities = ['Activity 1', 'Activity 2', 'Activity 3', 'Activity 4', 'Activity 5']
-        self.days = ['Monday','Tuesday','Wednesday','Thursday','Friday','Saturday','Sunday']
+        self.week = 1
+        self.activities = [['Activity 1', 'Activity 2', 'Activity 3', 'Activity 4', 'Activity 5']]
+        self.days = []
+        self.t = Timeline()
+        self.t.add_week()
+        self.days.append([])
+        
+        for i in range(len(self.t.timeline['week'+str(self.week)])):
```
Comment (midastown) [inline panel_grid.py:25]: line 25 to 27 will definitly have to move to the show_table() function that will get "a" week from the Timeline object

File: panel_grid.py [hunk 3]
```diff
@@ -46,34 +55,38 @@ def create_panel(self):
...
+                    label = Label(tab, text=self.activities[self.week-1][i-1])
                     label.grid(row=i, column=j)
                     tab.grid_columnconfigure(j, weight=1)
                 else:                  # adding the checkboxes
                     label = Checkbutton(tab) 
                     label.grid(row=i, column=j)
                     tab.grid_columnconfigure(j, weight=1)
         
-        # Oh shit there is no pack in this for loop at all, yeah man never
+        he = Button(tab,text='hello',command=self.add_week)
```
Comment (midastown) [inline panel_grid.py:78]: You see here the button need to be in it's own frame, like I said in the function definition, you should play with that Frame() to understand it.

**Review by midastown — APPROVED** [review]

**Pull request merged**